docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,45 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+The PyTorch framework is supported through export to the ONNX format. Model Optimizer Python API allows the conversion of PyTorch models using the `convert_model()` method, which internally converts a model to ONNX.
+
+## Converting a PyTorch model from memory with Python API (Experimental Functionality)
```

Comment:
Can we switch to PT FE path by default (as `use_new_frontend=True`) and keep old path via ONNX as a backup solution (as `use_new_frontend=False`)? This is more aligned with what we are aiming for in the long perspective. We can experiment with automatic fallback in case if PT FE pass failed with an appropriate message in the console (I think it is acceptable for feature that we are describing as experimental).

Reply:
There is a separate task for enabling PT FE in convert_model() https://github.com/openvinotoolkit/openvino/pull/15197
I suppose we can switch to PT FE by default when it is done. 

Should I write here that default path is PT FE?